patterns/package.json

```diff
@@ -5,7 +5,6 @@
     "babel-core": "^6.18.2",
     "babel-loader": "^6.2.8",
     "babel-preset-es2015": "^6.18.0",
-    "bower": "^1.8.0",
```

Comment:
We still require bower for some dependencies since some are not available on `NPM`.